packages/account/src/Sections/Security/DeactivateAccount/account-has-balance.jsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { Button, Icon, Money, ThemedScrollbars } from '@deriv/components';
+import { formatMoney } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+
+// TODO: [move-to-shared] - Remove the implementation in Client and add this to shared utils
+const getMT5AccountType = (group) => (group ? group.replace('\\', '_').replace(/_(\d+|master|EUR|GBP)/, '') : '');
+// TODO: [move-to-shared] - Remove the implementation in Client and add this to shared utils
+const getMT5AccountDisplay = (group) => {
+    if (!group) return {};
+
+    const value = getMT5AccountType(group);
+    let display_text = localize('MT5');
+    if (/svg$/.test(value)) {
+        display_text = localize('Synthetic');
+    } else if (/vanuatu/.test(value) || /svg_(standard|financial)/.test(value)) {
+        display_text = localize('Financial');
+    } else if (/labuan/.test(value)) {
+        display_text = localize('Financial STP');
+    }
+
+    return display_text;
+};
+
+const AccountHasBalanceOrOpenPositions = ({
+    accounts_with_balance_or_open_positions,
+    mt5_login_list,
+    client_accounts,
+    onBackClick,
+}) => {
+    const deriv_accounts_open_positions = [];
+    const deriv_accounts_with_balance = [];
+    const mt5_accounts_id = [];
+    const mt5_accounts_with_balance_or_open_positions = [];
+
+    if (accounts_with_balance_or_open_positions.open_positions) {
+        Object.keys(accounts_with_balance_or_open_positions.open_positions).forEach((account_name) => {
+            const deriv_account_with_open_position = client_accounts.find(
+                (client_account) => client_account.loginid === account_name
+            );
+            if (deriv_account_with_open_position) {
+                deriv_accounts_open_positions.push(deriv_account_with_open_position);
+            } else {
+                mt5_accounts_id.push(account_name);
+            }
+        });
+    }
+    if (accounts_with_balance_or_open_positions.balance) {
+        Object.keys(accounts_with_balance_or_open_positions.balance).forEach((account_name) => {
+            const is_deriv_account = client_accounts.find((client_account) => client_account.loginid === account_name);
+            if (is_deriv_account) {
+                const deriv_account = accounts_with_balance_or_open_positions.balance[account_name];
+                deriv_account.id = account_name;
+                deriv_accounts_with_balance.push(deriv_account);
+            } else {
+                mt5_accounts_id.push(account_name);
+            }
+        });
+    }
+    if (mt5_accounts_id.length !== 0) {
+        [...new Set(mt5_accounts_id)].forEach((account_name) => {
+            const mt5_account_with_balance = mt5_login_list.find((account_obj) => account_obj.login === account_name);
```

Comment:
and here:
```javascript
const mt5_account_with_balance = mt5_login_list.find(isActiveAccount);
```

Reply:
actually we can't use this helper here because in mt5 account object we have `login` as our key but in deriv accounts, we have `loginid` as our key.